### PR TITLE
fix(DB/Creature): fix damage modifier of Razormaw

### PR DESCRIPTION
**Pull request by elthehablo**
from 7.5 to 2.5. probably still a bit high. subject to change

## Changes Proposed:
-  damagemodifier from 7.5 to 2.5 (might be 2 or 1.5) but that would make it really easy

## Issues Addressed:

- Progress https://github.com/chromiecraft/chromiecraft/issues/5300
- Progress https://github.com/azerothcore/azerothcore-wotlk/issues/7819

## SOURCE:

https://youtu.be/9010SfVJPN0 but this shows damage to a pet. so ideally you should test this as a hunter with a pet at the same level. a cat pet, that is

## Tests Performed:

- tested myself as a paladin with appropriate gear at level 19. I take around 60ish damage each melee hit now, which is doable if it's a 2 man quest

## How to Test the Changes:

1. `.quest add 9689`
2. .tele bloodmyst
3. move to quest location
4. summon boss
5. see how much damage you take on melee, preferably as a hunter with a cat pet

## Known Issues and TODO List:

- [ the boss flies over the ground. it should walk, as seen in the video linked above ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Kitzunu** [comment]
Can this be sniffed?

**Comment by Gultask** [comment]
Sort of. Min and Max damages are sniffed (40-47 from with Multiplier 1.27 TBCmangos)
You need to calculate with unit_stats to figure out the Modifier

**Comment by acidmanifesto** [comment]
> Can this be sniffed?

It can be but im confused with the OP's subject to change comment.

**Comment by elthehablo** [comment]
my value of 2.5 is an estimate and should probably be fine-tuned is what I meant

File: data/sql/updates/pending_db_world/razormawdamageupdate.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- damage modifier edit of Razormaw from 7.5 to 1.36 based on mangos values
+
+UPDATE `creature_template` SET `DamageModifier` = 1.36 WHERE `entry` = 17592 AND `modelid1` = 17256;
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/razormawdamageupdate.sql:3]: Modelids are sniffed. Until probabilities get implemented we need to use scripting to change models instead, but this looks like leftover, though :p
```suggestion
UPDATE `creature_template` SET `DamageModifier` = 1.36 WHERE `entry` = 17592;
```

**Review by Gultask — CHANGES_REQUESTED** [review]

**Comment by Gultask** [comment]
I think I know the issue with the hovering. Let me see if we have sniffs for the waypoints

File: data/sql/updates/pending_db_world/razormawdamageupdate.sql (hunk 1 above)
Comment (heyitsbench) [reply data/sql/updates/pending_db_world/razormawdamageupdate.sql:3]: The model isn't getting changed here though, just being used in tandem with the entry for the WHERE section.

Comment (Gultask) [reply data/sql/updates/pending_db_world/razormawdamageupdate.sql:3]: Oh right, I didn't notice it :c 
It's not needed in this case

**Review by Gultask — APPROVED** [review]

**Pull request merged**